packages/logconsole-extension/src/index.tsx

```diff
@@ -83,16 +85,33 @@ function activateLogConsole(
   app: JupyterFrontEnd,
   rendermime: IRenderMimeRegistry,
   translator: ITranslator,
+  settingRegistry: ISettingRegistry,
+  toolbarRegistry: IToolbarWidgetRegistry,
```

Comment:
Some other JupyterLab plugins define `ISettingRegistry` and `IToolbarWidgetRegistry` as `optional`.

Maybe we should follow the same approach here too, for consistency?